core/controllers/suggestion.py

```diff
@@ -95,7 +96,8 @@ class SuggestionHandler(base.BaseHandler):
     @acl_decorators.can_suggest_changes
     def post(self):
         try:
-            suggestion = suggestion_services.create_suggestion(
+            suggestion = transaction_services.run_in_transaction(
```

Comment:
Maybe add a comment explaining why the transaction is needed.

Reply:
Done!

Reply:
Flagging that this caused a prod failure in production. We need to be much more careful in the future when introducing transactions.